linkerd/meshtls/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#![allow(irrefutable_let_patterns)]
```

Comment:
nit: we should make sure to remove this when we do add additional variants...

Reply:
No, we'll continue to want this so that things compile when there's only one variant enabled.

Reply:
ah, right, i didn't think that through...